### PR TITLE
Upgrade dependencies (including breaking ones) 

### PR DESCRIPTION
**Pull request by cjpatton**
~`serde-wasm-bindgen` is being upgraded, so we'll need to make sure we validate this change in Testing before merging downstream. This is marked as draft for now; I'll mark it as reviewable once I've tested it.~

EDIT: `serde-wasm-bindgen` is not being upgraded.

### CODE REVIEW

File: daphne_worker/src/durable/aggregate_store.rs [hunk 1]
```diff
@@ -374,7 +374,8 @@ impl AggregateStore {
                 js_sys::Reflect::set(
                     &chunks_map,
                     &JsValue::from_str(METADATA_KEY),
-                    &serde_wasm_bindgen::to_value(&meta)?,
+                    &serde_wasm_bindgen::to_value(&meta)
+                        .map_err(|e| worker::Error::RustError(e.to_string()))?,
```
Comment (mendess) [inline daphne_worker/src/durable/aggregate_store.rs:378]: Is this because `worker` depends on an earlier version of serde-wasm-bindgen? If that's the case it would be better to make a PR to them to bump `serde-wasm-bindgen` first or not bumping the version of `serde-wasm-bindgen` ourselves yet.

I checked what changed and it doesn't seem relevant to us

Comment (cjpatton) [reply daphne_worker/src/durable/aggregate_store.rs]: Done.

**Comment by cjpatton** [comment]
I've validated this in Testing.

**Review by mendess — APPROVED** [review]
